components/ProductListItem/index.js

```diff
@@ -1,25 +1,91 @@
 import styled from "styled-components";
 import Link from "next/link";
+import { useState } from "react";
+import { StyledSubmitButton, StyledCancelLink } from "../Buttons";
 
-const StyledLink = styled(Link)`
-  display: inline-block;
+const StyledListItem = styled.li`
   margin-block: 7px;
   width: 100%;
+  background-color: var(--secondaryBackgroundColor);
+  border-radius: 5px;
+  box-shadow: 0px 1px 3px var(--primaryDarkColor);
+`;
+
+const StyledLink = styled(Link)`
+  display: inline-block;
+  width: 85%;
   padding: 10px;
   text-decoration: none;
-  border-radius: 5px;
   color: var(--primaryDarkColor);
-  background-color: var(--secondaryBackgroundColor);
-  box-shadow: 0px 1px 3px var(--primaryDarkColor);
   &:hover {
     cursor: pointer;
   }
 `;
+const StyledDeleteIcon = styled.button`
+  width: 15%;
```

Comment:
No change required here, but i would be careful using % as for the width. This could lead to large differences in spacing between different screen sizes.

Reply:
Thanks for this warning / consideration 🙏